client/jetpack-connect/style.scss

```diff
@@ -1096,3 +1096,80 @@ body.is-section-jetpack-connect .layout.is-jetpack-woocommerce-flow {
 		margin-right: -5px;
 	}
 }
+
+// <ProductCard /> overrides
+.is-section-jetpack-connect {
+	.product-card {
+		@include breakpoint( '>660px' ) {
+			text-align: left;
+		}
+	}
+
+	.product-card__header {
+		@include breakpoint( '<660px' ) {
+			border-bottom: none;
+		}
+	}
+
+	.product-card__header-primary {
+		padding-top: 20px;
+		padding-bottom: 20px;
+	}
+
+	.product-card__header-secondary {
+		padding-top: 20px;
+		padding-bottom: 20px;
+
+		@include breakpoint( '<660px' ) {
+			padding-bottom: 8px;
+			border-top: solid 2px var( --color-jetpack-plan-free );
+		}
+
+		&::after {
+			bottom: auto;
+			top: 0;
+		}
+	}
+
+	.product-card__title,
+	.product-card__title:not( .is-purchased ) {
+		font-size: 20px;
+		color: var( --color-neutral-70 );
+
+		@include breakpoint( '>660px' ) {
+			font-size: 22px;
+			font-weight: 600;
+		}
+	}
+
+	.product-card__price-group {
+		justify-content: center;
+	}
+
+	.plan-price.is-discounted,
+	.plan-price.is-discounted .plan-price__currency-symbol,
+	.product-card__price-group.is-discounted .product-card__billing-timeframe,
+	.product-card__option .product-card__price-group.is-discounted .product-card__billing-timeframe {
+		color: var( --color-neutral-70 );
+	}
+
+	@include breakpoint( '>660px' ) {
+		.plan-price,
+		.plan-price__currency-symbol,
+		.plan-price__integer,
+		.plan-price__fraction,
+		.product-card__billing-timeframe {
+			font-size: 14px;
+			font-weight: 600;
+			font-style: normal;
+		}
+
+		.product-card__billing-timeframe {
+			color: var( --color-neutral-70 );
+		}
+	}
+
+	.product-card__option-name {
+		text-align: left;
```

Comment:
Does this needs to be here? 
Is it only specific to the Jetpack connection flow version?

Reply:
The Jetpack Connect page is in general centered in terms of horizontal alignment. We want the product card content to be centered in most of the cases, too.

However, for the product option name the `text-align: center` is unwanted and causes quirks in the layout:

![Screenshot 2019-10-21 at 12 11 04](https://user-images.githubusercontent.com/478735/67196866-e7a1eb00-f3fb-11e9-8203-e89a53842558.png)

That's why we have to explicitly call it out here.

Reply:
Thanks for the explanation!